docs/070_Configuration/020_Spring_Integration.md

```diff
@@ -0,0 +1,210 @@
+                    <property name="cooperativeThreadCount" value="2"/>
+                </bean>
+            </property>
+            <property name="defaultEdgeConfig">
+                <bean class="com.hazelcast.jet.config.EdgeConfig">
+                    <property name="queueSize" value="2048"/>
+                </bean>
+            </property>
+            <property name="properties">
+                <props>
+                    <prop key="foo">bar</prop>
+                </props>
+            </property>
+        </bean>
+    </constructor-arg>
+</bean>
+
+<bean id="map" factory-bean="instance" factory-method="getMap">
+    <constructor-arg value="my-map"/>
+</bean>
+```
+
+## Declaring Beans by _jet_ Namespace
+
+Hazelcast Jet has its own namespace ***jet*** for bean definitions. You can easily 
+add the namespace declaration `xmlns:jet=“http://www.hazelcast.com/schema/jet-spring”`
+to the beans element in the context file so that ***jet*** namespace shortcut can 
+be used as a bean declaration. Remember to add `hazelcast-jet-spring.jar` to classpath.
+
+Here is an example schema definition:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:jet="http://www.hazelcast.com/schema/jet-spring"
+       xmlns:hz="http://www.hazelcast.com/schema/spring"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+        http://www.hazelcast.com/schema/spring
+        http://www.hazelcast.com/schema/spring/hazelcast-spring-3.10.xsd
+        http://www.hazelcast.com/schema/jet-spring
+        http://www.hazelcast.com/schema/jet-spring/hazelcast-jet-spring-0.6.xsd">
+        <!-- ... -->
+ </beans>
+```
+#### Configuring Hazelcast Jet Instance 
+
+```xml
+<jet:instance id="instance">
+    <hz:config>
+        <hz:spring-aware/>
+        <hz:group name="jet"/>
+        <hz:network port="5701" port-auto-increment="false">
+            <hz:join>
+                <hz:multicast enabled="false"/>
+                <hz:tcp-ip enabled="true">
+                    <hz:member>127.0.0.1:5701</hz:member>
+                </hz:tcp-ip>
+            </hz:join>
+        </hz:network>
+        <hz:map name="map" backup-count="3">
+        </hz:map>
+    </hz:config>
+    <jet:instance-config cooperative-thread-Count="2"/>
+    <jet:default-edge-config queue-size="2048"/>
+    <jet:properties>
+        <hz:property name="foo">bar</hz:property>
+    </jet:properties>
+</jet:instance>
+```
+
+#### Configuring Hazelcsat Jet Client
+
+```xml
+<jet:client id="jet-client">
+    <jet:group name="jet"/>
+    <jet:network>
+        <hz:member>127.0.0.1:5701</hz:member>
+    </jet:network>
+    <jet:spring-aware/>
+</jet:client>
+```
+
+#### Hazelcast Jet Supported Type Configurations and Examples
+
+ - map
+ - list
```

Comment:
Why not Cache? ICache is supported accros Jet as well.

Reply:
Hazelcast IMDG does not provide direct access to the ICache as well instead exposes it with `CacheManager` interface and integrates it with Spring Cache Api. I've chosen the same approach for Jet.